apps/services/license-api/src/app/modules/license/test/license.service.spec.ts

```
@@ -223,6 +233,161 @@ export class MockUpdateClient extends BaseLicenseUpdateClient {
+      })
+    }
+    //some other error
+    return Promise.resolve<Result<PassVerificationData>>({
+      ok: false,
+      error: {
+        code: 99,
+        message: 'some service error',
+      },
+    })
+  }
+}
```

Biome

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)